src/utils/axiosSettings.js

```diff
@@ -10,19 +10,37 @@ export const getArticleList = () => {
   return thingsApi
     .get(urlString)
     .then(({ data }) => {
+      console.log(data);
       return data.articles;
     })
     .catch((error) => {
       console.log(error);
     });
 };
 
-const getArticle = (article_id) => {
+export const getArticle = (article_id) => {
   const urlString = `/articles/${article_id}`;
 
-  return thingsApi.get(urlString).then(({ data }) => {
-    return data;
-  });
+  return thingsApi
+    .get(urlString)
+    .then(({ data }) => {
+      return data;
+    })
+    .catch((error) => {
+      console.log(error);
+    });
 };
 
-export default getArticle;
+export const getComments = (article_id) => {
+  const urlString = `/articles/${article_id}/comments`;
+
+  return thingsApi
+    .get(urlString)
+    .then(({ data }) => {
+      console.log(data, "<<<data");
+      return data;
+    })
+    .catch((error) => {
+      console.log(error);
+    });
```

Comment:
Make sure to remove console logs before submitting pull requests. This isn't a biggie but something that is common in industry to good to get used to